context/brand-context/HISTORY.md

```diff
@@ -1,4 +1,10 @@
 # History
+## 16.0.0 (2021-10-06)
+    * BREAKING:
+        * Update the plus icon
+    * Add a minus svg icon
```

Comment:
This is actually breaking as well because it is `remove icon-minus.svg, add minus.svg`